### PR TITLE
fix(dev-infra): correct build setup for dev-infra

### PR DESCRIPTION
**Pull request by josephperrott**
Correct's the missing dependencies and incorrect build dependencies for the
dev-infra bazel targets.

### CODE REVIEW

File: dev-infra/tmpl-package.json [hunk 1]
```diff
@@ -12,11 +12,14 @@
     "@angular/benchpress": "0.2.1",
     "@octokit/graphql": "<from-root>",
     "@octokit/types": "<from-root>",
+    "@octokit/rest": "<from-root>",
     "brotli": "<from-root>",
     "chalk": "<from-root>",
     "cli-progress": "<from-root>",
+    "fs-extra": "<from-root>",
```
Comment (devversion) [inline dev-infra/tmpl-package.json:19]: I recently removed `fs-extra`. Where did we re-add it?

**Review by devversion — APPROVED** [review]
LGTM.

File: dev-infra/tmpl-package.json (hunk 1 above)
Comment (josephperrott) [reply dev-infra/tmpl-package.json:19]: It is currently used in `//dev-infra/utils/console.ts`.  If you would like to remove it, we can do so in follow up?

Comment (josephperrott) [reply dev-infra/tmpl-package.json:19]: Fixed as discussed offline.

**Pull request closed without merging**